src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageEndpoint.cs

```diff
@@ -33,22 +36,22 @@ internal class RazorLanguageEndpoint : IRazorLanguageQueryHandler
         {
             if (foregroundDispatcher == null)
             {
-                throw new ArgumentNullException(nameof(foregroundDispatcher));
+                throw new System.ArgumentNullException(nameof(foregroundDispatcher));
             }
 
             if (documentResolver == null)
             {
-                throw new ArgumentNullException(nameof(documentResolver));
+                throw new System.ArgumentNullException(nameof(documentResolver));
```

Comment:
I know I did this but just stating in this PR 😄 `using System`

Reply:
No I did this. Because System.Range and Blah.Blah.Omnisharp.Range conflicts.

Reply:
Lol yeah, I thought about changing it before but figured I'd mess with that code as little as possible.

Reply:
Didn't see @ajaybhargavb's comment, he's correct.

Reply:
o, lol instead of this just do `using Range = ...`

Reply:
Will do that in my PR. Waiting for this to be merged so I can rebase and add tests.